omero/users/history.rst

```diff
@@ -5,7 +5,7 @@ CHANGELOGS
 Links to decoupled repositories
 ===============================
 
-Since OMERO 5.5, the following repositories have been decoupled.
+Starting from OMERO 5.5, the following repositories have been decoupled.
```

Comment:
@jburel : this will need back porting to ome/openmicroscopy.

Reply:
yep I will do that

Reply:
actually this is not needed since the links are only in the doc and autogen append the history from openmicroscopy

Reply:
Not sure I follow but happy to try again with the next docs build. I definitely saw this commit being reverted though.

Reply:
https://github.com/ome/ome-documentation/pull/2089